regression/rt_ar_react_1x2v_quadraticV.c

```diff
@@ -0,0 +1,969 @@
+#include <math.h>
+#include <stdio.h>
+#include <time.h>
+
+#include <gkyl_alloc.h>
+#include <gkyl_const.h>
+#include <gkyl_fem_parproj.h>
+#include <gkyl_gyrokinetic.h>
+#include <rt_arg_parse.h>
+
+/*
+This is a test to see if our ionization/recombination operators produce coronal equilibrium
+Need at least 1e21 argon density otr distribution of charge states may be slightly different
+Need ne*time > 10^20
+
+*/
+
+struct gk_app_ctx {
+  double chargeElc; // electron charge
+  double massElc; // electron mass
+  double chargeIon; // ion charge
+  double massIon; // ion mass
+  double massAr; // Li mass
+  double Te; // electron temperature
+  double Ti; // ion temperature
+  double TAr; // Argon temperature
+  double c_s; // sound speed
+  double nuElc; // electron collision frequency
+  double nuIon; // ion collision frequency
+  double nuFrac;
+  double B0; // reference magnetic field
+  double n0; // reference density
+  double n0Ar; // argon reference density
+  double n0Ar1; // argon1 reference density
+  double n0Ar2; // argon2 reference density
+  double n0Ar3; // argon3 reference density
+  double n0Ar4; // argon4 reference density
+  double n0Ar5; // argon5 reference density
+  double n0Ar6; // argon6 reference density
+  double n0Ar7; // argon7 reference density
+  double n0Ar8; // argon8 reference density
+  double n0Ar9; // argon9 reference density
+  double n0Ar10; // argon10 reference density
+  double n0Ar11; // argon11 reference density
+  double n0Ar12; // argon12 reference density
+  double n0Ar13; // argon13 reference density
+  double n0Ar14; // argon14 reference density
+  double n0Ar15; // argon15 reference density
+  double n0Ar16; // argon16 reference density
+  double n0Ar17; // argon17 reference density
+  double n0Ar18; // argon18 reference density
+
+  // Simulation parameters
+  double Lz; // Box size in z.
+  double kperp; // perpendicular wave number used in Poisson solve
+  double n_src; // Source density.
+  double T_src; // Temp density.
+  double vpar_max_elc; // Velocity space extents in vparallel for electrons
+  double mu_max_elc; // Velocity space extents in mu for electrons
+  double vpar_max_ion; // Velocity space extents in vparallel for ions
+  double mu_max_ion; // Velocity space extents in mu for ions
+  double vpar_max_Ar; // Velocity space extents in vparallel for Li ions
+  double mu_max_Ar; // Velocity space extents in mu for Li ions  
+  double finalTime; // end time
+  int numFrames; // number of output frames
+};
```

Comment:
@jRoeltgen please make this test conform to the structure and logic in other tests (e.g. see rt_gk_sheath_3x2v). In particular, do what most other tests do to specify cdim, vdim, number of cells, temporal variables (e.g. num_frames, t_end, etc).